aws/container-linux/kubernetes/controllers.tf

```diff
@@ -30,6 +30,7 @@ resource "aws_instance" "controllers" {
   root_block_device {
     volume_type = "${var.disk_type}"
     volume_size = "${var.disk_size}"
+    iops        = "${var.disk_iops}"
```

Comment:
Begs the question of whether `iops` is a valid parameter for standard and gp2 disks and if so, what the value should be.